scripts/verify_verification_trigger_contract.py

```
@@ -0,0 +1,365 @@
+    scripts = set()
+    for line in guardrail_block:
+        stripped = line.strip()
+        if stripped.startswith("run: python3 scripts/"):
+            scripts.add(stripped.removeprefix("run: python3 ").strip())
+    return scripts
+
+
+def extract_just_recipe_deps(just_text: str, recipe: str) -> set[str]:
+    match = re.search(rf"^{re.escape(recipe)}:\s*(.*)$", just_text, re.MULTILINE)
+    if not match:
+        raise ValueError(f"justfile missing recipe '{recipe}'")
```

Ruff (0.15.9)

[warning] Avoid specifying long messages outside the exception class

(TRY003)